gnovm/pkg/test/test.go

```diff
@@ -415,6 +417,9 @@ func (opts *TestOptions) runTestFiles(
 		if rep.Failed {
 			err := fmt.Errorf("failed: %q", tf.Name)
 			errs = multierr.Append(errs, err)
+			if opts.FailfastFlag {
+				return errs
+			}
```

Comment:
Sorry, needs another round...

This won't stop execution for a subtest, ie. if I run a test using `t.Run` inside a test.

Reply:
What do you think of 79fa2289e3d5f1ae4c67c736c0422a9591efdfd4 ?